Installation/Docker.md

```diff
@@ -0,0 +1,346 @@
+---
+# icon: container
+label: Docker
+---
+
+# Docker Installation
+
+!!! info
+This guide assumes you installed SillyTavern in a non-root (non-admin) folder. If you installed SillyTavern in a root folder, you may have to run some of these commands with administrator rights [`sudo`, `doas`, Command Prompt (Administrator)].
+!!!
+
+## Installation
+
+### Linux
+
+1. Install Docker by following the Docker installation guide [here](https://docs.docker.com/engine/install/).
+   !!! danger
+   **Do not** install Docker Desktop.
+   !!!
+2. Follow the steps in **Manage Docker as a non-root user** in the Docker [Post-Installation Guide](https://docs.docker.com/engine/install/linux-postinstall/).
+3. Install [Git](https://git-scm.com/download/linux) using your package manager.
+
+    - Debian (Ubuntu/Pop! OS/etc.)
+
+        ```sh
+        sudo apt install git
+        ```
+
+    - Arch Linux (Manjaro/EndeavourOS/etc.)
+
+        ```sh
+        sudo pacman -S git
+        ```
+
+    - Fedora, Red Hat Enterprise Linux (RHEL), etc.
+        ```sh
+        sudo dnf install git
+        ```
+
+4. Clone the SillyTavern repository.
+
+    - Release (Stable Branch)
+
+        ```sh
+        git clone https://github.com/SillyTavern/SillyTavern && cd SillyTavern/docker
+        ```
+
+    - Staging (Development Branch)
+        ```sh
+        git clone https://github.com/SillyTavern/SillyTavern -b staging && cd SillyTavern/docker
+        ```
+
+5. Execute `docker compose` by running the following command within the Docker folder.
+
+    ```sh
+    docker compose up -d
+    ```
+
+6. Execute the following Docker command to obtain the IP of your SillyTavern Docker container.
+
+    ```sh
+    docker network inspect docker_default
+    ```
+
+    You should recieve some sort of output similar to the following below.
+
+    ```json
+    [
+        {
+            "Name": "docker_default",
+            "IPAM": {
+                "Config": [
+                    {
+                        "Subnet": "172.18.0.0/16",
+                        "Gateway": "172.18.0.1"
+                    }
+                ]
+            }
+        }
+    ]
+    ```
+
+    Copy down the IP you see in _Gateway_ as this will be important.
+
+7. Using `sudo`, open `nano` and run the following command.
+
+    ```sh
+    sudo nano config/config.yaml
+    ```
+
+    Within `nano`, go down to `whitelist`. You should see something similar to the following below.
+
+    ```yaml
+    whitelist:
+        - 127.0.0.1
+    ```
+
+    Add a new line below _127.0.0.1_ and put in the IP you copied from Docker. It should look something similar to the following afterwards.
```

Comment:
I would just note that if being run in docker, it's likely not the local machine that's connecting. Might be worth mentioning that you can also just whitelist as usual if it's on the default network / bridge / host.

Reply:
Thanks, added a note for that.